src/booleans.js

```diff
@@ -1,61 +1,65 @@
-function negate(a) {
-  // your code here
+const negate = (a) => {
+  return !(a);
```

Comment:
You dont need these brackets here, this works just as well:
```suggestion
  return !a;
```

This is the case for most of the return statements in this file